martin-tile-utils/src/lib.rs

```diff
@@ -204,6 +204,17 @@ pub fn tile_index(lon: f64, lat: f64, zoom: u8) -> (u32, u32) {
     (tile.x.min(max_value) as u32, tile.y.min(max_value) as u32)
 }
 
+#[must_use]
+#[allow(clippy::cast_possible_truncation)]
+#[allow(clippy::cast_sign_loss)]
+pub fn tile_colrow(lng: f64, lat: f64, zoom: u8) -> (u32, u32) {
+    let tile_size = EARTH_CIRCUMFERENCE / f64::from(1_u32 << zoom);
+    let (x, y) = wgs84_to_webmercator(lng, lat);
+    let col = ((x - (EARTH_CIRCUMFERENCE * -0.5)).abs() / tile_size).trunc() as u32;
+    let row = (((EARTH_CIRCUMFERENCE * 0.5) - y).abs() / tile_size).trunc() as u32;
```

Comment:
You don't need to call `trunc` here because that's exactly what `as u32` does - drops the floating point portion.

```suggestion
    let col = ((x - (EARTH_CIRCUMFERENCE * -0.5)).abs() / tile_size) as u32;
    let row = (((EARTH_CIRCUMFERENCE * 0.5) - y).abs() / tile_size) as u32;
```